dmoj/settings.py

```diff
@@ -161,6 +161,9 @@
 FONTAWESOME_CSS = '//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css'
 DMOJ_CANONICAL = ''
 
+DMOJ_ENABLE_COMMENTS = True
```

Comment:
Small nit, let's also move these up to below the rest of the `DMOJ_` constants/

Reply:
Done :)